components/PixelBlast.tsx

```
@@ -0,0 +1,739 @@
+        variant,
+        color,
+        speed,
+    ]);
+
+    return (
+        <div
+            ref={containerRef}
+            className={`relative h-full w-full overflow-hidden ${className ?? ''}`}
+            style={style}
+            aria-label="PixelBlast interactive background"
+        />
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)